dashboard/app/jobs.go

```diff
@@ -567,42 +569,41 @@ func createJobResp(c context.Context, job *Job, jobKey *db.Key) (*dashapi.JobPol
 // miss the situation when some actual user sends a patch testing request without
 // patch.
 func isRetestReproJob(job *Job, build *Build) bool {
-	return job.Type == JobTestPatch &&
+	return (job.Type == JobTestPatch || job.Type == JobBisectFix) &&
 		job.Patch == 0 &&
 		job.KernelRepo == build.KernelRepo &&
 		job.KernelBranch == build.KernelBranch
 }
 
-func handleRetestedRepro(c context.Context, now time.Time, job *Job, jobKey *db.Key, req *dashapi.JobDoneReq) error {
+func handleRetestedRepro(c context.Context, now time.Time, job *Job, jobKey *db.Key,
+	lastBuild *Build, req *dashapi.JobDoneReq) error {
 	bugKey := jobKey.Parent()
 	crashKey := db.NewKey(c, "Crash", "", job.CrashID, bugKey)
 	crash := new(Crash)
 	if err := db.Get(c, crashKey, crash); err != nil {
 		return fmt.Errorf("failed to get crash: %v", crashKey)
 	}
-	build, err := loadBuild(c, job.Namespace, crash.BuildID)
-	if err != nil {
-		return fmt.Errorf("failed to query build: %w", err)
-	}
-	if !isRetestReproJob(job, build) {
-		return nil
-	}
 	bug := new(Bug)
 	if err := db.Get(c, bugKey, bug); err != nil {
 		return fmt.Errorf("failed to get bug: %v", bugKey)
 	}
 	if !config.Namespaces[bug.Namespace].RetestRepros {
```

Comment:
Now we can get here with a fix bisection even if RetestRepros is false. Also for manual testing requests w/o a patch.
I think this condition can be removed.